convene-web/spec/spec_helper.rb

```diff
@@ -43,7 +43,9 @@
   # inherited by the metadata hash of host groups and examples, rather than
   # triggering implicit auto-inclusion in groups with matching metadata.
   config.shared_context_metadata_behavior = :apply_to_host_groups
-
+  config.define_derived_metadata do |meta|
```

Comment:
Not Blocking: We may want to add a comment explaining our reasoning for why we prefer to `aggregate_failures`.

I, personally, prefer that we `aggregate_failures` because without it, we can wind up in situations where we don't have a clear picture of what parts of the test are failing; which can make it hard to orient our understanding as we decide what actions to take.